genon/preprocessor/tests/unit/test_attachment_textloader_long_line_333.py

```
@@ -0,0 +1,115 @@
+
+
+@pytest.mark.unit
+@requires_weasyprint
+def test_long_single_line_txt_not_truncated(tmp_path: Path):
+    """줄바꿈 없는 긴 한 줄의 끝 텍스트가 청킹 결과에서 누락되지 않아야 한다(이슈 #333)."""
+    _get_pdf_path, TextLoader = _import_processor()
+
+    # A4 폭을 확실히 넘기는 긴 한 줄 + 문장 끝 고유 토큰.
+    # 수정 전에는 끝 토큰이 페이지 밖으로 밀려 렌더/추출에서 누락됨.
+    head = "교정시설 수용관리 업무 안내 " + ("가나다라마바사아자차카타파하 " * 40)
+    tail_token = "문장끝고유표식TAIL333"
```

Ruff (0.15.21)

[error] Possible hardcoded password assigned to: "tail_token"

(S105)